version_control_guidance.md

```diff
@@ -0,0 +1,17 @@
+# Version control guidance
+
+This guidance helps to explain how to apply version control to our code.
+
+It is compliant with our version control standards, but provides some additional information on ways to implement
+version control practice.
+
+## Guidance
+
+### Teams
+
+When managing access to version control repositories, it is good practice to create permission groups that
+represent wider teams grouped around an entire product or service rather than individual repositories or groups
+consisting of 'everyone'.
+
+This prevents individual repositories being used for multiple, distinct components that should each have their own
```

Comment:
Are you trying to say

> Avoid putting everything related to your project in a single repo. Put each component (apps, packages, libraries etc) in their own repos, and use teams to manage access to them.

If so I think it needs to be a little clearer. Not saying my version is better, but I think there is some ambiguity in the current version.